tools/maint/run_python_compiler.bat

```diff
@@ -2,8 +2,8 @@
 ::
 :: Automatically generated by `create_entry_points.py`; DO NOT EDIT.
 ::
-:: To make modifications to this file, edit `tools/run_python_compiler.bat` and
-:: then run `tools/create_entry_points.py`
+:: To make modifications to this file, edit `tools/maint/run_python_compiler.bat` and
+:: then run `tools/maint/create_entry_points.py`
```

Comment:
Can you also run this script so that `emcc.bat` and `em++.bat` are regenerated based on it?